[2023-03-23 15:17:25,501] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.trigger_defence manual__2023-03-23T15:16:48.608981+00:00 [queued]>
[2023-03-23 15:17:25,563] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.trigger_defence manual__2023-03-23T15:16:48.608981+00:00 [queued]>
[2023-03-23 15:17:25,566] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:17:25,574] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-03-23 15:17:25,576] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-23 15:17:25,663] {taskinstance.py:1377} INFO - Executing <Task(TriggerDagRunOperator): trigger_defence> on 2023-03-23 15:16:48.608981+00:00
[2023-03-23 15:17:25,680] {standard_task_runner.py:52} INFO - Started process 11933 to run task
[2023-03-23 15:17:25,690] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'trigger_defence', 'manual__2023-03-23T15:16:48.608981+00:00', '--job-id', '2970', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmpa3kxgrdo', '--error-file', '/tmp/tmpymwc0rlq']
[2023-03-23 15:17:25,694] {standard_task_runner.py:80} INFO - Job 2970: Subtask trigger_defence
[2023-03-23 15:17:25,952] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.trigger_defence manual__2023-03-23T15:16:48.608981+00:00 [running]> on host 6447f1dacd32
[2023-03-23 15:17:26,383] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=trigger_defence
AIRFLOW_CTX_EXECUTION_DATE=2023-03-23T15:16:48.608981+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-23T15:16:48.608981+00:00
[2023-03-23 15:17:32,611] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=trigger_defence, execution_date=20230323T151648, start_date=20230323T151725, end_date=20230323T151732
[2023-03-23 15:17:32,767] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-23 15:17:32,942] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
